src/endless/res/endless.loc

```diff
@@ -265,6 +265,7 @@ t MSG_339 "%s (Recommended)"
 t MSG_340 "%s (Maximum)"
 t MSG_341 "You currently have %s free on your %s %s drive."
 t MSG_342 "Select how much of your %s drive to dedicate to Endless OS."
+t MSG_345 "Select how much of your %s drive to dedicate to Endless Code."
 
 t MSG_350 "Unfortunately, there is not enough storage space."
 t MSG_351 "To continue installing Endless OS, please free up at least %s by deleting videos, photos, or other large files on your %s drive."
```

Comment:
One option would be to take a leaf out of [Rufus' book](https://github.com/endlessm/rufus/blob/master/src/localization.c#L409) and use one of the high bits to mark strings as Code-flavoured; then have some `uprintf` wrapper which, if in Code mode, checks whether a high-bit variant of a string exists and uses it instead. I don't think it's worth the complexity for just a few strings, but maybe if the number grows...